.github/workflows/markdown_link_check.yml

```diff
@@ -15,5 +15,4 @@ jobs:
       with:
         use-quiet-mode: 'yes'
         use-verbose-mode: 'yes'
-        check-modified-files-only: 'yes'
```

Comment:
We probably want to keep this 'yes' right?

Reply:
We can keep it, but we will miss that check if a link dies for another reason unrelated to a code change.

Reply:
Right, I see. Deleting it is best then.